### ISSUE TITLE
Disney Infinity series - Graphical issue with blocky artifacts

### ISSUE THREAD

**Issue by ChwaraeTeg** [RPCS3/rpcs3#14572]
### Quick summary

Graphics appear glitched in the Disney Infinity games

### Details

In the Disney Infinity games (Disney Infinity, Disney Infinity 2.0 and Disney Infinity 3.0) the graphics will appear with blocky artifacts all over both during gameplay and during cutscenes (but not in menus/loading screens). The games seem to require WCB+RCB+WDB+RDB to avoid appearing overly bright, but even with these settings on, the block effect still appears.

RPCS3 Example from Infinity 2.0:
![DisneyInfinity2_RPCS3](https://github.com/RPCS3/rpcs3/assets/104720038/429269e2-fba0-4884-95a5-88d0257c7e95)

Console Example:
![DisneyInfinity2_PS3](https://github.com/RPCS3/rpcs3/assets/104720038/7bc89ff4-69af-453b-b8cb-7dd08706c784)

### Attach a log file

Log File: [DisneyInfinity2.log.gz](https://github.com/RPCS3/rpcs3/files/12456490/DisneyInfinity2.log.gz)

### Attach capture files for visual issues

RSX Capture: https://mega.nz/file/bNcwjSya#0Rl0T1JjGNzwJYkIZLAwrSxTcXp8LpUqo_zMVuYUMOw
Renderdoc Capture: https://mega.nz/file/GF9HhJwB#y0mcZz49aUfMVS_44p9Svo720OUlLgkZoNEu3gWEVqA

### System configuration

Intel Xeon CPU E3-1240 v5 @ 3.50GHz | 16 GiB RAM | Operating system: Windows, Major: 10, Minor: 0, Build: 19045 | NVIDIA GeForce GTX 1060 6GB running on driver 537.13.0.0

### Other details

As well as enabling WCB+RCB+WDB+RDB, the game also seemed to run more stable with FIFO Accuracy set to Atomic, so this was enabled in the above example. Tested with Strict Rendering Mode enabled also which made no difference, and the same issue was present using OpenGL instead of Vulkan.

**Comment by ChwaraeTeg** [comment]
With 'Handle RSX Memory Tiling' enabled, this is much improved in v0.0.29-15694-23cb67e0 but there is still some graphical glitches present:

![Infinity](https://github.com/RPCS3/rpcs3/assets/104720038/1a2ad98b-b037-4505-a877-dd9667ca11a5)

Log: [Infinity2_HandleRSXMemoryTiling.log.gz](https://github.com/RPCS3/rpcs3/files/13063089/Infinity2_HandleRSXMemoryTiling.log.gz)
RSX Capture: https://mega.nz/file/zUlzwTDS#fiCfvgZtc_y72o0PtmjobGKhO47Shi1Z34zIms0a_vA

**Comment by digant73** [comment]
@ChwaraeTeg is that still an issue?

**Comment by kd-11** [comment]
Should still be broken. The hardware-accelerated detiler has some edge cases with corner slices that I never fixed.

**Comment by Jack54guythecoder** [comment]
is this still an issue?

**Comment by Sambro09** [comment]
> is this still an issue?

Yes this is still an issue. I just booted up as well on 3.0 and having the same visual glitches.

**Comment by Rezzy-dev2** [comment]
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/162a6cb5-76b5-4987-9275-541c2d974b78" />

It still has a lot of rendering/shader bugs that are slowing down its performance, but Disney Infinity 1.0 actually displays mostly fine, and is playable. (Can't say the same about 2.0 and 3.0 -- for which the above detiler issues apply.)

Settings needed:

In addition to the graphics settings recommended above, Vsync should be turned to "Full", and the "Strict Rendering Mode" option should be enabled, otherwise the game freezes up when loading new areas.

RPCS3 Version: 0.0.41-19525-f5e8a2bf Alpha | master
Graphics card: NVIDIA Corporation AD107M [GeForce RTX 4060 Max-Q / Mobile]
System: Linux Mint 22.1 Cinnamon 6.4.8, X11 Window System, Linux Kernel version: 6.8.0-117-generic
Memory: 15.2 GiB RAM
